build/vs/BrainaliveDevices.cpp

```diff
@@ -0,0 +1,167 @@
+/****************************************************************************
```

Comment:
This file needs to live in Engine/Devices

Reply:
@dioptre as I already mentioned in my previous comment please refer to [Add BLE Functionality (#100)](https://github.com/neuromore/studio/pull/100) In this PR I already move these files into the Engine/Devices. but now you closed (#100) PR so I resubmit the PR.